internal/pkg/views/notifications.go

```diff
@@ -112,11 +112,14 @@ func (w *NotificationWidget) ConfirmDelete() {
 			Timeout:    time.Second * 15,
 		})
 
-		swaggerExpander := expanders.GetSwaggerResourceExpander()
-
 		for _, i := range pending {
-			swaggerDeleted, err := swaggerExpander.Delete(context.Background(), i)
-			if err == nil && !swaggerDeleted {
+			var err error
+			fallback := true
+			if i.Expander != nil {
+				deleted, err := i.Expander.Delete(context.Background(), i)
```

Comment:
general comment: should we be handling contexts a bit better here? Using timeouts and allowing cancellations - something for another PR.

Reply:
Yeah, that's probably worth raising an issue for!